aspnetcore/fundamentals/host/web-host.md

```diff
@@ -15,7 +15,7 @@ ASP.NET Core apps configure and launch a *host*. The host is responsible for app
 
 ::: moniker range=">= aspnetcore-3.0"
 
-This article covers the Web Host, which remains available only for backward compatibility. The [Generic Host](xref:fundamentals/host/generic-host) is recommended for all app types.
+This article covers the Web Host, which remains available only for backward compatibility.The ASP.NET Core templates create a .NET Core Generic Host (<xref:Microsoft.Extensions.Hosting.HostBuilder>). Most apps should use the [Generic Host](xref:fundamentals/host/generic-host). `Generic Host` is recommended for all app types.
```

Comment:
I think the last two lines might have conflicted with each other. One said "most apps should use" and the other said "recommended for all apps". Maybe something more like this?:

```suggestion
This article covers the Web Host, which remains available only for backward compatibility. The ASP.NET Core templates create a [.NET Generic Host](<xref:fundamentals/host/generic-host>), which is recommended for all app types.
```